test_pytest_timeout.py

```diff
@@ -410,3 +416,27 @@ def test_foo():
     if child.isalive():
         child.wait()
     assert b"Timeout >1.0s" not in result
+
+
+@have_spawn
+@have_ipdb
+def test_suppresses_timeout_when_ipdb_is_entered(testdir):
+    pytest.importorskip("pexpect")
```

Comment:
Doesn't this file already unconditionally import pexecpt?  When should this line trigger?

Reply:
You're right. I copied it from the previous test not really knowing what it was doing but it would appear that that isn't needed.